### PR TITLE
[FEATURE] Add a command to set up TYPO3

### PR DESCRIPTION
**Pull request by linawolf**
references https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/283

### CODE REVIEW

File: Documentation/Installation/Install.rst [hunk 1]
```diff
@@ -81,8 +97,40 @@ will contain the following structure:
...
--------------------
+Run the setup process
+=====================
+
+Setup TYPO3 in the console
+--------------------------
+
+..  versionadded:: 12.1
+    Starting with TYPO3 12.1 a new CLI command `setup` is introduced as
+    an alternative to the existing GUI based web installer.
```
Comment (brotkrueml) [inline Documentation/Installation/Install.rst:108]:
```suggestion
    an alternative to the existing GUI-based web installer.
```

File: Documentation/Installation/TutorialDdev.rst [hunk 2]
```diff
@@ -97,6 +97,22 @@ You now have a **Composer-based TYPO3 installation**.
 Run the Installation Setup Tool
 -------------------------------
 
+Setup TYPO3 in the console
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  versionadded:: 12.1
+    Starting with TYPO3 12.1 a new CLI command `setup` is introduced as
+    an alternative to the existing GUI based web installer.
```
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:105]:
```suggestion
    an alternative to the existing GUI-based web installer.
```

**Review by brotkrueml — COMMENTED** [review]
I am missing the description of the automated setup with environment variables as outlined in the feature rst. Is there a reason to omit it?

**Comment by linawolf** [comment]
> I am missing the description of the automated setup with environment variables as outlined in the feature rst. Is there a reason to omit it?

That is not really a getting started topic?

**Comment by brotkrueml** [comment]
> That is not really a getting started topic?

But somewhere it must be documented? Do we have another chapter about setup?

**Comment by linawolf** [comment]
I don't think the topic fits here. we already describe to many different options on how to setup a TYPO3 system. all of this is very confusing for newcommers. We also until now do not describe all console commands. I would move this ticket from `resolves` to `references` and we can find a better place for advanced usage.

**Comment by brotkrueml** [comment]
Agreed, I would expect one place where I can find everything about the installation. Then the according "Getting Started" chapter about installation is only a kickstarter and can be reduced - with links to the "one place to rule them all".

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**